packages/web/src/scss/components/Grid/README.md

```diff
@@ -183,6 +183,37 @@ All nested Grid Items have their CSS variables firstly reset to the default valu
 This allows you to nest Grids with Grid Items and set their column values independently.
 Please note that every, even nested, Grid has 12 columns if not specified otherwise.
 
+### Grid Placement Props: `--grid-item-column-start`, `--grid-item-column-end`, `--grid-item-row-start`, `--grid-item-row-end`
+
+These props control the placement of grid items within a CSS Grid layout by specifying where the item starts and ends in rows and columns:
+
+- `--grid-item-column-start`: Specifies the vertical grid line where the item begins.
+- `--grid-item-column-end`: Specifies the vertical grid line where the item ends. The item spans up to, but does not include, this grid line. For example, if columnEnd is "3", the item ends at the line before the third column.
+- `--grid-item-row-start`: Specifies the horizontal grid line where the item begins.
+- `--grid-item-row-end`: Specifies the horizontal grid line where the item ends. The item spans up to, but does not include, this grid line. For example, if rowEnd is "4", the item ends at the line before the fourth row.
+
+#### Grid Placement Example
+
+```html
+<div
+  class="GridItem"
+  style="
+      --grid-item-column-start: 2;
+      --grid-item-column-end: 5; <!-- or 'span 3' -->
```

Comment:
Nitpicking – Because the value is not set with single quotes, but you would like to distinguish the value via formatting, maybe you could be more descriptive:

```suggestion
      --grid-item-column-end: 5; <!-- or --grid-item-columns-end: span 3; -->
```

?